examples/podman/README.md

```diff
@@ -0,0 +1,29 @@
+
+# Podman Examples
+
+These examples demonstrate using samba container images with podman.
+
+## Domain Joined SMB Shares
+
+Included here is a shell script and a JSON config template that
+demonstrates using podman to run an smbd instance joined to Active Directory.
```

Comment:
I think we might make it a little more clear that this examples requires an existing AD domain that the winbindd container will actually join with the credentials provided in the script.

Reply:
Sure. Will do.

Reply:
Done.